tests/drivers/dma/mux_req_gen/prj.conf

```diff
@@ -0,0 +1,6 @@
+CONFIG_TEST=y
+CONFIG_DMA=y
+CONFIG_DMAMUX_STM32=y
```

Comment:
This should not be needed.
Symbol DMAMUX_STM32 should be enabled if DMA is true on SoCs supporting it

Reply:
removed